DESCRIPTION

```diff
@@ -32,7 +32,9 @@ Imports:
     rlang (>= 1.0.0),
     rstudioapi (>= 0.5),
     tools,
-    yaml (>= 2.1.5)
+    yaml (>= 2.1.5),
+    RcppTOML,
```

Comment:
We don't seem to be importing from these new dependencies, should we? 

Alternatively, could they instead go in `Suggests` to make them optional?

Reply:
@nealrichardson - We had discussed this and decided to take these as `Imports` in an attempt to avoid additional friction when folks publish.